saleor/settings.py

```diff
@@ -671,7 +671,13 @@ def get_host():
 
 EXTENSIONS_MANAGER = "saleor.extensions.manager.ExtensionsManager"
 
+PAYMENT_PLUGINS = [
+    "saleor.payment.gateways.dummy.plugin.DummyGatewayPlugin",
+    "saleor.payment.gateways.stripe.plugin.StripeGatewayPlugin",
+    "saleor.payment.gateways.braintree.plugin.BraintreeGatewayPlugin",
+]
 PLUGINS = os.environ.get("PLUGINS", [])
+PLUGINS += PAYMENT_PLUGINS
```

Comment:
This can be problematic when someone by mistake adds payment plugins to `PLUGIN` from env.
What do you think?

Reply:
I think we could filter duplicates here but the best way to that would probably be to prevent same plugin from being loaded more than once in manager.

Reply:
Also, it is getting a string, not a list 😉